packages/lib/src/components/Riverty/Riverty.tsx

```diff
@@ -20,15 +22,27 @@ export default class Riverty extends OpenInvoiceContainer {
         ...OpenInvoiceContainer.defaultProps
     };
 
+    constructor(props) {
```

Comment:
Do we need this constructor? It doesn't do anything special.

Reply:
Well, spotted. We don't it was just there because at some point it was needed during dev.